### ISSUE TITLE
Use node to download mongodb rather than execSync() + curl

### ISSUE THREAD

**Issue by vkarpov15** [vkarpov15/run-rs#43]

**Comment by ucejtech** [comment]
Node.js fs module could do this, would that be a better implementation?

**Comment by vkarpov15** [comment]
Yes definitely. We haven’t gotten around to this yet, ran into some roadblocks so we just used curl

**Comment by ucejtech** [comment]
Would you hint at what roadblocks to look out for, I would be trying to fix this issue .

**Comment by vkarpov15** [comment]
I think I mostly struggled with figuring out how to get something from nodes http request that I could pipe into an fs write stream. Since then I wrote a tutorial about making http requests with node, so this may be helpful: https://masteringjs.io/tutorials/node/http-request
